datahub-classify/src/datahub_classify/infotype_predictor.py

```diff
@@ -55,6 +58,16 @@ def predict_infotypes(
             # get the configuration
             config_dict = global_config[infotype]
 
+            # convert exclude_name list into a set for o(1) checking
+            if EXCLUDE_NAME in config_dict and config_dict[EXCLUDE_NAME] is not None:
+                config_dict[EXCLUDE_NAME] = (
+                    set(config_dict[EXCLUDE_NAME])
+                    if not strip_exclusion_formatting
```

Comment:
Currently, the customer we're working with may want the ability to have the `EXCLUDE_NAME` list only pertain to exact matches. The `strip_exclussion_formatting` flag would allow them to decide if they only want to use exact matches, or if they want both exact matches and matches after stripping special chars & case changes.

Does that make sense? Please let me know if there is flaw in my logic.

Reply:
Okay, so if one of the customer needs is an exact match, then the _strip_exclussion_formatting_ flag seems appropriate.